### PR TITLE
feat: Add commonly used unique id generators

### PR DESCRIPTION
**Pull request by nimerritt**

### CODE REVIEW

File: test-data-helper.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+function randomStringFromAlphabet(length, alphabet) {
+  let str = '';
+  for (let i = 0; i < length; i++) {
+    str += alphabet.charAt(Math.floor(Math.random() * alphabet.length));
+  }
+  return str;
+}
+
+function uniqueObjectId() {
+  return new ObjectId();
```
Comment (dbrockman) [inline test-data-helper.js]: - [x] This function can probably be removed

File: test-data-helper.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+function toObjectId(object_id_string) {
+  return new ObjectId(object_id_string);
```
Comment (dbrockman) [inline test-data-helper.js]: - [x] remove this one as well

File: test-data-helper.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+function uniqueObjectIdString() {
+  return uniqueObjectId().toString();
```
Comment (dbrockman) [inline test-data-helper.js]:
``` js
const crypto = require('crypto');

function uniqueObjectIdString() {
  return crypto.randomBytes(12).toString('hex');
}
```

File: test-data-helper.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+function uniquePrefixedString(prefix) {
+  return prefix + uniqueObjectIdString();
+}
+
+function uniqueEmail() {
+  const id = new ObjectId();
```
Comment (dbrockman) [inline test-data-helper.js]: - [x] replace `new ObjectId()` with `uniqueObjectIdString()`

**Review by dbrockman — CHANGES_REQUESTED** [review]

**Comment by dbrockman** [comment]
- [x] add a note in the readme about `require('@springworks/test-harness/test-data-helper')`

File: test-data-helper.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+function uniqueDeviceId() {
+  return randomIMEI();
+}
+
+function randomVin() {
+  return uniquePrefixedString('vin-');
+}
+
+function randomRegNumber() {
```
Comment (dbrockman) [inline test-data-helper.js:62]: Maybe we should rename this to `randomSwedishRegNumber`?

Comment (nimerritt) [reply test-data-helper.js:62]: I agree, but I want to minimise the refactoring of tests when using this package. Perhaps in the future, or with a argument with default value?

**Review by dbrockman — APPROVED** [review]

**Pull request merged**